### PR TITLE
Axios thoughts

### PR DESCRIPTION
**Pull request by tobiasqueck**
@kranthie-sap @ullasholla as a follow up to our discussion, here is how I'd prefer the ADT code to be abstracted.

### CODE REVIEW

File: packages/axios-extension/src/abap/abap-service-provider.ts [hunk 1]
```diff
@@ -47,9 +46,7 @@ export class AbapServiceProvider extends ServiceProvider implements AbapServiceP
     public async getAtoInfo(): Promise<AtoSettings> {
```
Comment (kranthie-sap) [inline packages/axios-extension/src/abap/abap-service-provider.ts:46]: Should this also belong to the ADT service?

**Review by kranthie-sap — CHANGES_REQUESTED** [review]
Looks good to me. There's one comment about `getAtoInfo`.
Also, it looks like the pipeline failed due to linting errors.

File: packages/axios-extension/src/abap/abap-service-provider.ts (hunk 1 above)
Comment (slin-sap) [reply packages/axios-extension/src/abap/abap-service-provider.ts:46]: Yes. This is also ADT service.

**Comment by tobiasqueck** [comment]
This PR/branch is obsolete. Some of the proposed changes with further improvements have been applied by @slin-sap in other PRs

**Pull request closed without merging**